.github/workflows/docs-build-push.yml

```diff
@@ -19,6 +19,7 @@ on:
       - "docs/**"
 
 permissions:
+  pull-requests: write
```

Comment:
It's best if default permissions on the workflow are `contents: read`.  Then add the permissions needed on the job.  If I remember correct, this was feedback from the scorecard security job.